src/main/java/com/commercetools/paymenttoorderprocessor/jobs/actions/MessageReader.java

```diff
@@ -127,7 +135,20 @@ private void fetchUnprocessedMessagesFromPlatform() {
     private MessageQuery buildQuery() {
 
         MessageQuery messageQuery = MessageQuery.of()
-                .withPredicates(m -> m.type().is(MESSAGETYPE))
+                .plusPredicates(m -> {
```

Comment:
Why did you change it?

Reply:
Also I find it useful to prefer `.plusPredicates` unless you really want to overwrite predicate. The reason is I could have multiple predicates over the method and in the future if I want to move the order of the predicates, I could easily forget to change `.withPredicates` to `.plusPredicates` and then it would not work properly anymore. This mistake I saw in other PRs that people forgot to change `.withPredicates` to `.plusPredicates`.